sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/Capabilities.java

```diff
@@ -32,9 +32,9 @@ public boolean supportsGpuResource() throws IOException {
         return hasOrExceedsVersion(1, 9);
     }
 
-    public boolean supportCniPortMapping() throws IOException {
+    public boolean supportsCNINetworking() throws IOException {
         // CNI port-mapping is supported by DC/OS 1.10 upwards
-        return hasOrExceedsVersion(1, 10);
+        return hasOrExceedsVersion(1, 9);
```

Comment:
Is 1.9 the correct value? (I don't know either way). If so should probably update the comment..